### ISSUE TITLE
Failed to load OptiX library [Clang-9 / Nvidia 440.26 / Cuda 10.1 / OptiX 6.5]

### ISSUE THREAD

**Issue by Awcrr** [mitsuba-renderer/mitsuba2#39]
First of all, congratulations to the grand release! Have been waiting for mitsuba2 for a while.

Based on the extensive install instructions, I successfully installed the "gpu_autodiff_rgb" variant. Unfortunately, I couldn't run the simple cbox example as provided [in the doc](https://mitsuba2.readthedocs.io/en/latest/src/inverse_rendering/diff_render/#enumerating-scene-parameters). 

The error message says 

```
rt_check(): OptiX API error = 1546 (Failed to load OptiX library! Very likely, your NVIDIA graphics driver is too old and not compatible with the version of OptiX that is being used. In particular, OptiX 6.5 
requires driver revision R435.80 or newer.) in ../src/librender/scene_optix.inl:40.
```

which i think should not be the case since I double-checked that I have proper versions of OptiX and the Nvidia driver. Can you please give me some hints about what might go wrong in this case? Thanks!

**Comment by Speierers** [comment]
Is this on Windows?
Could you try to add `/path/to/optix/bin64/` and `/path/to/optix/lib64/` to your `PATH`?

**Comment by Awcrr** [comment]
No it's on Linux/Debian. I just added the `path/to/optix/bin` and `path/to/optix/lib64` to my `PATH` and also I have `path/to/optix/lib64` in my `LD_LIBRARY_PATH`. Unfortunately, it shows the same error. Is there any suggested way that I can try to diagnose the issue? Thanks!

**Comment by Awcrr** [comment]
At last, I managed to detect the problem. It seems that I installed a very weird version of NVIDIA driver and optix which lack some libraries, e.g. `libnvoptix.so` files. After re-installing them, I resolved the problem. One take-away for others is try to use `strace` to check if all the dependencies are in place and in the `PATH`/`LD_LIBRARY_PATH`.

Thanks for your help Speierers! I think we can close this issue.
